features/partners.feature

```diff
@@ -10,38 +10,121 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
     # N.B. Je kan ontbonden/beëindigde huwelijken of partnerschappen raadplegen met de Haal-Centraal-BRP-historie-bevragen API.
 
     Scenario: Actueel huwelijk
-      Gegeven de persoon met burgerservicenummer 999990184 heeft de volgende huwelijken/partnerschappen in de registratie
-        | Categorie | soort verbintenis (15.10) | Voornamen (02.10) | Datum aangaan (06.10) | Datum ontbinding (07.10) | Onjuist (84.10) |
-        | 5         | H                         | Merel             | 20010215              |                          |                 |
-      Als de persoon met burgerservicenummer 999990184 wordt geraadpleegd met fields=partners
-      Dan bevat het antwoord een partner met naam.voornamen "Merel"
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999990184 |
+      En de persoon heeft een partner met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999992935 |
+      En de partner heeft de volgende naam gegevens
+      | naam              | waarde |
+      | voornamen (02.10) | Merel  |
+      En de partner heeft de volgende aangaanHuwelijkPartnerschap gegevens
+      | naam                      | waarde   |
+      | datum aangaan (06.10)     | 20010215 |
+      | soort verbintenis (15.10) | H        |
+      | indicatie onjuist (84.10) |          |
```

Comment:
het voor deze rule belangrijkste is nu niet genoemd: de partner heeft geen ontbinding huwelijk partnerschap gegevens